src/www/ui/api/Models/FileInfo.php

```diff
@@ -85,11 +85,11 @@ public function getJSON()
   public function getArray()
   {
     return array(
-      'view_info' => $this->view_info,
-      'meta_info' => $this->meta_info,
-      'package_info' => $this->package_info,
-      'tag_info' => $this->tag_info,
-      'reuse_info' => $this->reuse_info
+      'viewInfo' => $this->view_info,
+      'metaInfo' => $this->meta_info,
+      'packageInfo' => $this->package_info,
+      'tagInfo' => $this->tag_info,
+      'reuseInfo' => $this->reuse_info
```

Comment:
I would instead recommend to make the model version aware. See https://github.com/fossology/fossology/pull/2617/files#diff-f62f1d2b860c504c87d386cdfa79513c109848593f0afeb60a5dc76e4b98ff42 for reference.